packages/ibm-products/src/components/SidePanel/SidePanel.tsx

```diff
@@ -61,51 +187,54 @@ export let SidePanel = React.forwardRef(
 
       actionToolbarButtons,
       actions,
-      animateTitle = defaults.animateTitle,
+      animateTitle = true,
```

Comment:
Didn’t notice this in previous TypeScript PRs, but interesting to see that as part of the migration, we switch to referencing the default values more directly now. 👀

Reply:
Yes, it was causing a typescript error which I can't remember now what it was. I can try reverting it in this PR to see if we can go back to our previous approach.

Reply:
There were some formatting changes that have been applied from other files. But I've added the default props back similar to how we've included them previously.